src/stores/pacbioLibraries.js

```diff
@@ -0,0 +1,122 @@
+import { defineStore } from 'pinia'
```

Comment:
I think it would be good to provide some clearer documentation to the jsdoc standard?